src/utils/api/constants.ts

```diff
@@ -1,4 +1,5 @@
 // TODO: Make this configurable based on prod/dev run status
+// export const BASEURL = "https://nodes.jup.io/nxt?"; // prod URL
 export const BASEURL = "http://localhost:3080/nxt?"; // proxy runs on 3080 by default
```

Comment:
we can define a PROD vs DEV variable that the vitejs server will replace here. probably a safer way than assuming we had it set properly for a delivery.

Reply:
not a high priority for me atm but I wouldn't stop you from doing it